build.cake

```diff
@@ -0,0 +1,270 @@
+    .Does(() =>
+    {
+        CleanDirectories(new DirectoryPath[] { outputTests });
+    });
+
+Task("RunTests")
+  .Description("Executes xUnit tests")
+  .WithCriteria(!skipTests)
+  .IsDependentOn("CleanTests")
+  .Does(() =>
+{
+    var projects = GetFiles("./Hyperion.Tests/*.xproj");
+  //   - GetFiles("./tests/**/*.Performance.xproj");
+  // var projects = GetFiles("./tests/**/*.xproj")
+  //   - GetFiles("./tests/**/*.Performance.xproj");
+
+    foreach(var project in projects)
+    {
+        if (IsRunningOnWindows()) {
+            DotNetCoreTest(project.GetDirectory().FullPath, new DotNetCoreTestSettings
+            {
+                Configuration = configuration,
+                NoBuild = true,
+                Verbose = false,
+                ArgumentCustomization = args =>
+                  args.Append("-xml").Append(outputTests.Path.CombineWithFilePath(project.GetFilenameWithoutExtension()).FullPath + ".xml")
+            });
+        }
+        else {
+            var name = project.GetFilenameWithoutExtension();
+            var dirPath = project.GetDirectory().FullPath;
+            var xunit = GetFiles(dirPath + "/bin/" + configuration + "/net452/*/dotnet-test-xunit.exe").First().FullPath;
+            var testfile = GetFiles(dirPath + "/bin/" + configuration + "/net452/*/" + name + ".dll").First().FullPath;
+
+            using(var process = StartAndReturnProcess("mono", new ProcessSettings{ Arguments = xunit + " " + testfile }))
+            {
+                process.WaitForExit();
+                if (process.GetExitCode() != 0)
+                {
+                    throw new Exception("Mono tests failed!");
+                }
+            }
+        }
+    }
+});
+
+
+
+Task("Copy-Files")
+    .Does(() =>
+{
+    // .NET 4.5
+    DotNetCorePublish("./Hyperion", new DotNetCorePublishSettings
+    {
+        Framework = "net45",
+        OutputDirectory = outputBinariesNet45,
+        Configuration = configuration,
+        NoBuild = true,
+        Verbose = false
+    });
+
+    // .NET Core
+    DotNetCorePublish("./Hyperion", new DotNetCorePublishSettings
+    {
+        Framework = "netcoreapp1.0",
+        OutputDirectory = outputBinariesNetstandard,
+        Configuration = configuration,
+        NoBuild = true,
+        Verbose = false
+    });
+
+    // Copy license
+    CopyFileToDirectory("./LICENSE", outputBinariesNet45);
+    CopyFileToDirectory("./LICENSE", outputBinariesNetstandard);
+});
+
+
+Task("Package-NuGet")
+  .Description("Generates NuGet packages for each project that contains a nuspec")
+  .Does(() =>
+{
+    var projects = GetFiles("./src/**/*.xproj");
+    foreach(var project in projects)
+    {
+        DotNetCorePack(project.GetDirectory().FullPath, new DotNetCorePackSettings {
+            Configuration = configuration,
+            OutputDirectory = outputNuGet
+        });
+    }
+
+    // Cake - .NET 4.5
+    NuGetPack("./Hyperion/nuspec/Hyperion.nuspec", new NuGetPackSettings {
```

Comment:
We don't need nuspec files in NetCore projects, We could use project.json for that purposes